pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -81,6 +81,28 @@ type BindingInfo struct {
 	pv *v1.PersistentVolume
 }
 
+// StorageClassName returns the name of the storage class.
+func (b *BindingInfo) StorageClassName() string {
+	return b.pv.Spec.StorageClassName
+}
+
+// VolumeResource represents volume resource.
+type VolumeResource struct {
+	Requested int64
+	Capacity  int64
+}
+
+// VolumeResource returns volume resource.
+func (b *BindingInfo) VolumeResource() *VolumeResource {
```

Comment:
`VolumeResource` does not quite match the semantics here, also it's a term already used here:

https://github.com/kubernetes/kubernetes/blob/703e2bd2ebae1cc7737c12344a9e6fb13a6019a9/test/e2e/storage/framework/volume_resource.go#L43

Can we rename it to be more self-descriptive?

Reply:
renamed it to StorageResource, which is aligned to the "compute resource".